core/src/main/scala/org/apache/gearpump/cluster/ClusterMessage.scala

```diff
@@ -58,7 +58,7 @@ object ClientToMaster {
 
   case class QueryAppMasterConfig(appId: Int)
 
-  case class QueryHistoryMetrics(appId: Int, path: String)
+  case class QueryHistoryMetrics(appId: Int, path: String, readLatest: Boolean = false)
```

Comment:
If there is a data type Period or duration, I'd suggest change the method to `case class QueryMetrics(appId: Int, path: String, period: Period = null)`